wallet-test/src/test/scala/org/bitcoins/wallet/util/GetAddresses.scala

```diff
@@ -0,0 +1,102 @@
+package org.bitcoins.wallet.util
+
+import org.bitcoins.core.hd._
+import play.api.libs.json._
+import scala.sys.process._
+
+/** This program connects to a running Trezor, and gets
```

Comment:
A running trezor, or a running emulator? Is there no difference?

Reply:
There's no difference from `trezorctl`s point of view